### ISSUE TITLE
UserAddressBook: Using the last argument as keyword parameters is deprecated

### ISSUE THREAD

**Issue by captproton** [solidusio/solidus#3765]
When I migrate, I get this:
```
% rake db:migrate
/Users/carltanner/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solidus_core-2.10.2/app/models/concerns/spree/user_address_book.rb:8: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```
**Solidus Version**:
2.10

**To Reproduce**

1. create a new rails app (in my case Rails 6.0.3.3)
2. add to Gemfile:
```
gem 'solidus'
gem 'solidus_auth_devise'
```

3. bundle

4. rake db:reset

5. migrate
```
 % rake db:migrate
/Users/carltanner/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/solidus_core-2.10.2/app/models/concerns/spree/user_address_book.rb:8: warning: Using the last argument as keyword parameters is deprecated; maybe ** should be added to the call
```

Oh, and I am doing this on my Catalina Mac.

**Comment by stefano-sarioli** [comment]
Hi @captproton, Ruby 2.7 introduced a lot of warning, the one you are seeing is the most common.

This is been fixed and it will disappear with Solidus 2.11.
If you are interested you can have a look at the PRs the fix the problems (#3746 #3737 #3742)

Thanks anyway for taking the time to open this issue 🙏 .

**Comment by jarednorman** [comment]
As @stefano-sarioli pointed out, this fixed in master, but thanks for the reporting!
